src/client/src/services/model/trade.js

```diff
@@ -23,7 +24,7 @@ export default class Trade {
     tradeDate:Date,
     valueDate:string,
     status:TradeStatus,
-    isNew:boolean
+    isStateOfTheWorld:boolean
```

Comment:
I know this was already here but I think it's modeled incorrectly, the fact that trade is part of the initial batch has nothing to do with trade economics. 

Really this state should exist in a wrapper payload